### DISCUSSION TITLE
Sign in with multiple users

### DISCUSSION THREAD

**Discussion by milespop** [Soluto/oidc-server-mock#152] [Q&A]
Hi team, 

Just wondering how I would go about including a custom `oid` property for each user in my `USERS_CONFIGURATION_INLINE`

I've included it `oid` in the AllowedScopes property for the `CLIENTS_CONFIGURATION_INLINE` but I still don't see the oid property in the access token granted.

Heres an example of my **USERS_CONFIGURATION_INLINE**:
```
      USERS_CONFIGURATION_INLINE: |
        [  
          {
              "SubjectId": "1",
              "Username": "admin",
              "Password": "admin",
              "Claims": [
                  {
                    "Type": "profile",
                    "Value": "{\"roles\": [\"Administrator\"] }",
                    "ValueType": "json"
                  },
                  {
                      "Type": "oid",
                      "Value": "example-oid",
                      "ValueType": "string"
                  },
               ...(omitted for brevity)
              },
              ...(omitted for brevity)
          ]              
```

**Comment by AleF83** [comment]
Hi,
I guess that the missing part is to declare a scope, associate the scope with the custom claim and to request this scope in the authentication request.

Search `some-app-user-custom-claim` or `some-custom-identity-user-claim` in E2E tests for example.
